doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,255 @@
+plot_comparison(mask_2, mask_open, "mask before", "after opening")
+
+#####################################################################
+# Since "opening" an image starts with an erosion operation, bright regions
+# which are smaller than the structuring element have been removed.
+# Let us try now with a larger footprint:
+
+footprint = morphology.diamond(3)
+mask_open = morphology.opening(mask_2, footprint)
+plot_comparison(mask_2, mask_open, "mask before", "after opening")
+
+#####################################################################
+# Next, we can make the detected areas wider by applying a dilation filter:
+
+mask_dilate = morphology.dilation(mask_open, footprint)
+plot_comparison(mask_open, mask_dilate, "before", "after dilation")
+
+#####################################################################
+# Dilation enlarges bright regions and shrinks dark regions.
+# Notice how, indeed, the white spots have thickened.
+
+#####################################################################
+# Inpaint each frame separately
+# =============================
+# We are now ready to apply inpainting to each frame. For this we use function
+# ``inpaint_biharmonic`` from the ``restoration`` module. It implements an
+# algorithm based on biharmonic equations.
+# This function takes two arrays as inputs:
+# The image to restore and a mask (with same shape) corresponding to the
+# regions we want to inpaint.
+
+image_seq_inpainted = np.zeros(image_seq.shape)
+
+for i in range(image_seq.shape[0]):
+    image_seq_inpainted[i] = restoration.inpaint_biharmonic(
+        image_seq[i],
+        mask_dilate
+    )
+
+#####################################################################
+# Let us visualize one restored image, where the dark spots have been
+# inpainted. First, we find the contours of the dark spots (i.e., of the mask)
+# so we can draw them on top of the restored image:
+
+contours = measure.find_contours(mask_dilate)
+
+# Gather all (row, column) coordinates of the contours
+x = []
+y = []
+for contour in contours:
+    x.append(contour[:, 0])
+    y.append(contour[:, 1])
+# Flatten them
+x_flat = np.concatenate(x).ravel()
+y_flat = np.concatenate(y).ravel()
+# Create a sparse matrix corresponding to the mask of these contours
+data = np.ones(x_flat.shape, dtype='bool')
+mtx = sparse.coo_matrix((data, (x_flat, y_flat)), shape=mask_dilate.shape)
+# Convert it to array
+arr = mtx.toarray().astype('int')
+# Pick one frame
+sample_result = image_seq_inpainted[12]
+# Normalize it (so intensity values range [0, 1])
+sample_result /= sample_result.max()
+
+#####################################################################
+# We use function ``label2rgb`` from the ``color`` module to overlay the
+# restored image with the segmented spots, using transparency (alpha
+# parameter).
+
+color_contours = color.label2rgb(
+    arr,
+    image=sample_result,
+    alpha=0.4,
+    bg_label=0
```

Comment:
```suggestion
    bg_color=(1, 1, 1)
```
because it looks better with a transparent white background:
![bg_color](https://github.com/scikit-image/scikit-image/assets/2227806/a0ee6b11-4ec5-456c-8153-31f1d4662a3f)
than with a transparent black background:
![bg_label](https://github.com/scikit-image/scikit-image/assets/2227806/95dc010e-d1d5-4f5d-bf7d-4e4c0f699958)

I would actually prefer a 'None' background, but it doesn't seem readily available, contrary to what the docstring says: https://github.com/scikit-image/scikit-image/blob/49a98547e909697f6ee89ec70691c7cad7eeb823/skimage/color/colorlabel.py#L156-L157

Reply:
I'll commit this, since it seems obvious that we want to be able to *see* something... Begging for your upvotes @ana42742 and @decorouz :wink:

Reply:
Committed in 564ea6a1714a79eff633d964e4a2f5fa9dffe2f7.